tests/publiccloud/slem_basic.pm

```diff
@@ -114,6 +116,26 @@ sub run {
     $instance->run_ssh_command(cmd => 'sudo sestatus | grep enabled');
     $instance->run_ssh_command(cmd => 'sudo dmesg');
     $instance->run_ssh_command(cmd => 'sudo journalctl -p err');
+
+    # volume size tests
+    if (get_var('PUBLIC_CLOUD_ROOT_DISK_SIZE')) {
+        my $size = get_var('PUBLIC_CLOUD_ROOT_DISK_SIZE');
```

Comment:
shouldn't we use `get_required` here ?

Reply:
I believe we have jobs where we don't specify the disk size and just leave the defaults.

Reply:
> shouldn't we use `get_required` here ? Otherwise it could be empty.

if isnt defined ., its simply skipped :D